src/Analyzer.Core.JsonRuleTests/README.md

```diff
@@ -0,0 +1,46 @@
+# Test Project for Built-in JSON Rules
+This project is designed for quickly adding tests to verify the correctness of built-in JSON rules.  When new rules are added, tests should be added here to make sure the rule is written correctly and help protect against incorrect changes to the rule.
+
+## Test Setup
+A test (or tests) for a given JSON rule consists of 2 parts:
```

Comment:
I feel like this description can be reworded to be clearer. When I read this intro section, I didn't follow without seeing an example of a test file at the same time and didn't realize that the second point was the templates in the TestTemplates folder until I got the end of this README...